Rakefile

```diff
@@ -35,6 +35,9 @@ task :web do
     config[:schedule] = sched
   end
 
+  class DoWork
```

Comment:
When job is enqueued, Sidecloq needs existence class for determination that it is a kind of active_job or not. So I added this lines for prevent error that happen at web "Enqueue Now" action. What kind of solution should I fix?